final_task/rss_reader/requirements.txt

```diff
@@ -0,0 +1 @@
+html2text
```

Comment:
As far as i know `dateutil` is not a standard library, so it should be specified here

Reply:
Sorry I overlooked it, will fix